compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenInterfaces.td

```diff
@@ -269,6 +269,19 @@ def IREECodegen_LoweringConfigAttrInterface :
         return ::llvm::SmallVector<int64_t>();
       }]
     >,
+    InterfaceMethod<
+      /*desc=*/[{
+        Returns the number of tiling levels of the configuration. Returns -1 if
+        it is unknown.
+      }],
+      /*retTy=*/"int64_t",
```

Comment:
Not sure what the consensus/guideline is in IREE, but I generally prefer `std::optional` instead of using a sentinel value? And then you could make it unsigned as well?